.gitignore

```diff
@@ -29,8 +29,10 @@ geonode/static/lib/css/multi-select.css
 geonode/static/lib/css/qunit.css
 geonode/static/lib/css/select2.css
 geonode/static/lib/css/assets.min.css
+geonode/static/lib/css/leaflet-plugins.min.css
 geonode/static/lib/css/bootstrap-tokenfield.css
 geonode/static/lib/js/assets.min.js
+geonode/static/lib/js/leaflet-plugins.min.js
```

Comment:
Included to follow .gitignore pattern that minified js and css should not automatically be committed, because it will change for every `grunt production`